python/foglamp/tasks/statistics/__main__.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Purge process starter"""
+
+import sys
+
+from foglamp.tasks.statistics.statistics_history import StatisticsHistory
+from foglamp.common import logger
+
+__author__ = "Terris Linenbach, Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+if __name__ == '__main__':
```

Comment:
as `__main__.py` is package entry point, we can avoid `if __name__ == '__main__':`